corehq/apps/linked_domain/tests/test_linked_apps.py

```diff
@@ -232,8 +233,9 @@ def test_overrides(self):
         self.assertEqual(self.plain_master_app.logo_refs, {})
         self.assertEqual(self.linked_app.linked_app_logo_refs, logo_refs)
         self.assertEqual(self.linked_app.logo_refs, logo_refs)
-        self.assertEqual(self.linked_app.target_commcare_flavor, 'commcare_lts')
+        self.assertEqual(self.linked_app.commcare_flavor, 'commcare_lts')
         self.assertEqual(self.linked_app.linked_app_attrs, {
+            # ToDo: check why is this even a linked app attr? its not overridden
```

Comment:
It tests that if the master and linked app have different values for flavor, the linked app's doesn't get overridden when it's pulled. See https://github.com/dimagi/commcare-hq/pull/23964

Reply:
aha, thanks.. i confused it with something else